docker-compose.yml

```diff
@@ -910,3 +986,4 @@ volumes:
   mysql57_data:
   mysql80_data:
   redis_data:
+  enterprise_catalog_mysql8:
```

Comment:
This won't be used, because this isn't linked with `mysql80` container. These IDAs should use default devstack db volume so no need to add this